app/controllers/insured/consumer_roles_controller.rb

```diff
@@ -327,6 +327,33 @@ def help_paying_coverage_response
 
   private
 
+  def validate_person_match
+    first_name = params[:person][:first_name]
+    last_name = params[:person][:last_name]
+    return unless (first_name != @person.first_name) || (last_name != @person.last_name)
```

Comment:
@vkghub I think the way it is originally actually works better, because let's say a consumer only makes changes to their last name, we want to be able to perform the match query.